macros/JsSidebar.ejs

```diff
@@ -28,10 +29,11 @@ var text = mdn.localString({
     'Guide_OOP': 'Details of the object model',
     'Guide_Iterators_Generators': 'Iterators and generators',
     'Guide_Meta': 'Meta programming',
-    'Introductory': 'Introductory',
+    'Complete_beginners': 'Complete beginners',
     'Basics': 'JavaScript basics',
-    'Overview': 'JavaScript technologies overview',
-    'OO_JavaScript': 'Introduction to Object Oriented JavaScript',
+    'First_steps': 'JavaScript first steps',
+    'Building_blocks': 'JavaScript Building blocks',
```

Comment:
I meant the lower case b, which I think is correct according to our style guide.